### DISCUSSION TITLE
Mini PCIE Zwave Board

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js#6922] [General]
FYI on a new combined ZigBee + Zwave 800 board and external antennas:
https://www.mixtile.com/store/accessory/mixtile-2-in-1-zigbee-z-wave-mpcie-interface-module/

I ordered one to play with since I have a  number of mini PCs with Mini PCIe slots.

I have no association with that company, just thought others may be interested.

**Comment by AG-Teammate** [comment]
Anyone with external antenna can share their experience?

**Reply by Botched1** [reply]
I tried the z-wave.me Z-Station which has external antennas. Signal was SIGNIFICANTLY higher/better than the UZB7 I normally use. Every RSSI value was 10+ dB better. I stopped using it though as it uses proprietary firmware and backup/restore.

This mini-pcie device in the OP uses standard chips/firmware so is more attractive to me. I'll report back once I get it and test it (I think shipping is a month+ out).

**Reply by Botched1** [reply]
Just a follow-up on this, the shipping date is now listed as July 12. Will report back after I get the couple I ordered and get a chance to play with them.
